app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>Contestants</h1>
+<hr>
+<% @contestants.each do |contestant| %>
+  <section class='contestant' id='contestant-<%= contestant.id %>'>
+    <h3><%= "#{contestant.name}" %></h3>
+    <p>Projects:
+      <br>
+      <% @con_proj.each do |con_proj| %>
```

Comment:
You may be able to grab all of the associated projects with `contestant.projects` and then loop over that collection.